### DISCUSSION TITLE
Exception: module 'ansys.motorcad.core' has no attribute 'is_running_in_internal_scripting'

### DISCUSSION THREAD

**Discussion by Fthalizade** [ansys/pymotorcad#399] [Q&A]
Dear All,
I need to create Trapezoidal ducts of stator in Ansys Motor Cad software. I tried  Pymotorcad code. But it returns: Exception: module 'ansys.motorcad.core' has no attribute 'is_running_in_internal_scripting' . Can you help me please?

**Comment by JackB-Ansys** [comment]
Hello, please ensure you are using the latest version of Motor-CAD and PyMotorCAD.
